### PR TITLE
feat(individual-transcript-page): Transcript Metadata Card

### PR DESCRIPTION
**Pull request by 0tuedon**
##  What to note

- To test this component I have created a temporary route "/test-component" or [preview here](https://registry-git-fork-0tuedon-transcript-d2a57c-btc-knowledge-base.vercel.app/test-component) You will be able to see the component used
 
-
-  BDP-UI components that were not present were created into the ```component/svgs``` path. and this would be removed once the BDP UI components have been updated

-  I have closed this  https://github.com/bitcointranscripts/registry/pull/13

### CODE REVIEW

**Comment by kouloumos** [comment]
The information that are displayed in this component we are calling them "metadata", so please use this language where applicable. So this component should be "TranscriptMetadataCard", and actually this is the name that @Emmanuel-Develops called it when created the related issue: https://github.com/bitcointranscripts/registry/issues/10

**Comment by 0tuedon** [comment]
> #10

Thanks @kouloumos , I wasn't tagged to the issue and used the name I saw on figma
Screenshot 2024-09-09 at 10 29 59

**Comment by kouloumos** [comment]
> Thanks @kouloumos , I wasn't tagged to the issue and used the name I saw on figma Screenshot 2024-09-09 at 10 29 59


I understand the confusion. I wasn’t aware of the specific names on Figma either, so I’ll review them and update the names as needed. Also, @Emmanuel-Develops is working on creating an issue for each task so that issues can serve as the single source of truth moving forward.

File: src/app/components/transcripts/TranscriptBioData.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+              <p className="text-base md:text-lg font-semibold">Topic</p>
+            </div>
+            <div className="flex flex-wrap gap-2">
+              {topics &&
+                topics.map((topic) => (
+                  <p
+                    key={topic}
+                    className=" max-content py-[4.11px] px-[16.43px] rounded-[5.13px] bg-gray-custom-700 max-md:px-3 max-md:py-[2px] max-xl:text-[13px] max-md:text-sm max-md:border max-md:border-gray-custom-300 max-md:leading-[100%]"
+                  >
+                    {topic}
```
Comment (kouloumos) [inline src/app/components/transcripts/TranscriptBioData.tsx:78]: Topics are clickable and redirect to the specific topic's page

File: src/app/components/transcripts/TranscriptBioData.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            </div>
+            <div className="flex flex-wrap gap-2">
+              {speakers &&
+                speakers.map((speaker) => (
+                  <p
+                    key={speaker}
+                    className=" max-content py-[4.11px] px-[16.43px] rounded-[5.13px] bg-gray-custom-700 max-md:px-3 max-md:py-[2px] max-xl:text-[13px] max-md:text-sm max-md:border max-md:border-gray-custom-300 max-md:leading-[100%]"
+                  >
+                    {speaker}
+                  </p>
```
Comment (kouloumos) [inline src/app/components/transcripts/TranscriptBioData.tsx:97]: Speakers are clickable and redirect to the specific topic's page

**Comment by Emmanuel-Develops** [comment]
Apologies for the delay in review @0tuedon. The plan was to work on the transcript page next, priorities have shifted unfortunately to explore page, and this component will not be used until then.

Transcript metadata will be put on hold as missing pieces are yet to be built in the explore page.
issue: #17

**Comment by kouloumos** [comment]
@0tuedon should we close this one?

**Pull request closed without merging**